frontend/app/api/data-collection/[id]/points/route.ts

```
@@ -0,0 +1,377 @@
+      summary.totalCount = dataPoints.reduce((sum, dp) => {
+        const value = typeof dp.value === 'number' ? dp.value : (dp.value?.count || 0);
+        return sum + value;
+      }, 0);
+      summary.averagePerSession = summary.totalCount / dataPoints.length;
+      break;
+
+    case 'DURATION':
+      // Calculate total and average duration
+      const durations = dataPoints
+        .map(dp => dp.duration || 0)
+        .filter(d => d > 0);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)